### ISSUE TITLE
Certain XmlSerializer constructors are not supported in NetNative

### ISSUE THREAD

**Issue by khdang** [dotnet/corefx#7991]
The following tests fail in NetNative due the corresponding constructors are not supported on NetNative.
```
XmlSerializerTests.Xml_ConstructorWithXmlRootAttr
XmlSerializerTests.Xml_ConstructorWithXmlAttributeOverrides
```

**Comment by zhenlan** [comment]
Potentially this can be solved by using reflection based serialization in XmlSerializer in the future.

**Comment by huanwu** [comment]
Fixed with #10903
